app/assets/stylesheets/shared/tooltips.scss

```diff
@@ -0,0 +1,77 @@
+.tooltip-item {  
+  $base-border-color: #dcdcdc !default;
+  $base-border-radius: 3px !default;
+  $base-line-height: 1.5em !default;
+  $dark-gray: #333 !default;
+  $base-font-color: $dark-gray !default;
+  $tooltip-background: #ffffff;
+  $tooltip-border: transparent;
+  $tooltip-color: $base-font-color;
+  $tooltip-max-width: 16em;
+  $tooltip-arrow-width: 8px;
+  $tooltip-shadow: 0 2px 2px #c0c0c0;
```

Comment:
Color literals like `#c0c0c0` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#c0c0c0` should be written as `#C0C0C0`